protocol/protocol.go

```diff
@@ -95,6 +95,8 @@ const (
 	OpUnseal Op = 0x22
 	// OpRPC executes an arbitrary exported function on the server.
 	OpRPC Op = 0x23
+	// OpCustom requests a custom operation that can be defined by a function set in the server configuration
+	OpCustom Op = 0x24
```

Comment:
This doesn't seem very extensible to me. I'd prefer to see this named something like `OpCustom1` so it's clear this has a single purpose, or to add another tag to identify what kind of custom operation is being performed.

Reply:
Agree, I will go the route of adding a function name to implementation mapping to the configuration of the opcode, and expect that function name in the payload.